vic/drivers/python/src/display_current_settings.c

```diff
@@ -414,11 +414,11 @@ display_current_settings(int mode)
         fprintf(LOG_DEST, "STATEYEAR\t\t%d\n", global_param.stateyear);
         fprintf(LOG_DEST, "STATEMONTH\t\t%d\n", global_param.statemonth);
         fprintf(LOG_DEST, "STATEDAY\t\t%d\n", global_param.stateday);
-        if (options.BINARY_STATE_FILE) {
-            fprintf(LOG_DEST, "BINARY_STATE_FILE\tTRUE\n");
+        if (options.STATE_FORMAT) {
```

Comment:
Not sure why you are doing this again, since you just did this in l.398 and following